lib/go/edgecontext/edgecontext_test.go

```diff
@@ -240,6 +242,97 @@ func TestNew(t *testing.T) {
 	}
 }
 
+func TestLocale(t *testing.T) {
+	t.Run(
```

Comment:
🔕 the code would be cleaner if you use table test:

```go
for _, c := range []struct{
  label string
  locale string
  valid bool
}{
  {
    label: "valid-language",
    locale: "es",
    valid: true,
  },
  //...
} {
  t.Run(c,label, func(t *testing.T) {
    err := EdgeContextWithLocaleCode(c.locale)
    if (err == nil) != c.valid {
      t.Errorf(...)
    }
  })
}
```

Reply:
Definitely cleaner, updated in https://github.com/reddit/edgecontext/pull/12/commits/ca3ecb78969b7173ea01c490af773764ed983626